src/templates/next/types/LocalSearchInputBox.ts

```diff
@@ -0,0 +1,5 @@
+export interface LocalSearchInputBoxProps {
```

Comment:
confused why there's `NavbarLocalSearchProps` and there's this also (same for searchSG)
is it better to move the `type` here, get rid of the NavbarXXX types and then do the discriminated union on these two types instead?

Will have to move these types out of /next also

Reply:
Yeah good point, adjusted in 6c387b237453581a1443fe6f589bcc337429a394!